package.json

```diff
@@ -40,26 +40,29 @@
     "typescript": ">=2.4.0"
```

Comment:
Don't need that anymore. Might want to rebase the PR on master to avoid extra changes.